CMakeLists.txt

```diff
@@ -352,11 +352,16 @@ find_package(GLUT)
 find_package(Qt4 4.6)
 find_package(PLplot)
 
+if (USE_MKL)
+    set(GADGETRON_BUILD_RPATH "/opt/intel/mkl/lib/intel64:/opt/intel/lib/intel64_lin/")
```

Comment:
Please don't use hardcoded paths. MKL comes in a dozen different flavors and users may have odd install paths. 
We're already finding MKL explicitly - would the MKL_LIBRARY_DIRS work?